### PR TITLE
Add travis merge fuction

### PR DESCRIPTION
**Pull request by SchSeba**
- Add language
- Add golang version
- Add empty script

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/ovs-cni/45)

### CODE REVIEW

**Comment by SchSeba** [comment]
@phoracek Can you take a look?

File: .travis.yml [hunk 1]
```diff
@@ -9,15 +10,20 @@ env:
 - BUILDER_NAME="ovs-cni/${TRAVIS_JOB_ID}builder"
 
 install: true
+script:
+  - cd ../..
+  - mv kubevirt ../kubevirt.io
+  - cd ../kubevirt.io/ovs-cni
+  - make docker-build
```
Comment (phoracek) [inline .travis.yml]: Don't you need to pass TAG variable when building?

You either can move this line to deploy section or better, run `docker tag` in the deploy section.

Comment (SchSeba) [reply .travis.yml]: Good one!

File: .travis.yml [hunk 2]
```diff
@@ -9,6 +10,10 @@ env:
 - BUILDER_NAME="ovs-cni/${TRAVIS_JOB_ID}builder"
 
 install: true
+script:
+  - cd ../..
+  - mv kubevirt ../kubevirt.io
+  - cd ../kubevirt.io/ovs-cni
 
 deploy:
 - provider: script
```
Comment (phoracek) [inline .travis.yml]: don't you need to `cd` to the project folder? `make docker-build` triggers `make build`.

Comment (phoracek) [reply .travis.yml]: Could you please try it on your environment? This looks error prone.

Comment (phoracek) [reply .travis.yml]: I meant trying it on your repo and your Travis. If it is not a big problem

**Pull request merged**